lib/identity_config.rb

```diff
@@ -145,6 +145,8 @@ def self.build_store(config_map)
     config.add(:aws_kms_client_contextless_pool_size, type: :integer)
     config.add(:aws_kms_client_multi_pool_size, type: :integer)
     config.add(:aws_kms_key_id, type: :string)
+    config.add(:aws_kms_multi_region_key_id, type: :string)
+    config.add(:aws_kms_multi_region_write_enabled, type: :boolean)
```

Comment:
Do you want to add a spec for this being enabled/disabled, or does that come later?

Reply:
That will come later